engine/buildkit/client.go

```diff
@@ -483,51 +490,20 @@ func (c *Client) WriteStatusesTo(ctx context.Context, recorder *progrock.Recorde
 	}()
 }
 
-func (c *Client) RegisterClient(clientID, clientHostname, secretToken string) error {
```

Comment:
These methods for setting/checking secret tokens are now on `DaggerServer` (I'm not sure why I ever put them on `buildkit.Client` in the first place tbh)

Reply:
Confirmed this should be good; did some cleanup to use `WithoutCancel` too in this commit https://github.com/dagger/dagger/pull/6806/commits/e7e7235556bec84308c635007d12d3dfc78e577e